packages/backend/src/apps/custom-api/actions/http-request/index.ts

```diff
@@ -75,5 +75,7 @@ export default defineAction({
     }
 
     $.setActionItem({ raw: { data: responseData } })
+
+    return {}
```

Comment:
To repeat comments just in case - we can technically return `null` here to make it look better, but that infers to `Promise<any>` which feels kinda dangerous...